dmscripts/helpers/auth_helpers.py

```diff
@@ -17,12 +26,43 @@ def get_auth_token(api, stage):
     if not auth_token:
         DM_CREDENTIALS_REPO = os.environ.get('DM_CREDENTIALS_REPO', '../digitalmarketplace-credentials')
         token_prefix = 'J' if subprocess.check_output(["whoami"]) == 'jenkins' else 'D'
-        creds = subprocess.check_output([
-            "{}/sops-wrapper".format(DM_CREDENTIALS_REPO),
-            "-d",
-            "{}/vars/{}.yaml".format(DM_CREDENTIALS_REPO, stage)
-        ])
-        auth_tokens = yaml.safe_load(creds)[api]['auth_tokens']
+        creds_json = _decrypt_yaml_file_with_sops(DM_CREDENTIALS_REPO, "vars/{}.yaml".format(stage))
+        auth_tokens = creds_json[api]['auth_tokens']
         auth_token = next(token for token in auth_tokens if token.startswith(token_prefix))
 
     return auth_token
+
+
+def _get_nested_value(creds_json, env_var_path_list):
```

Comment:
Some tests or a docstring for this would give good impression of the sort of args it's expecting/ will work with.

Reply:
@benvand Pushed a commit 👍 (Also moved the top level script into the `framework-applications` folder, to live with the other framework lifecycle scripts)

Reply:
@katstevens I think the latest commit copied the file rather than moving it?

Reply:
🤦‍♀ Will sort that out